themes/cardinal_service/pdb_components/opportunities_list/src/Components/SelectList.jsx

```diff
@@ -149,6 +149,7 @@ export const SelectList = ({defaultValue, field, label, multiple, onChange, opti
       <Autocomplete
         classes={myStyles}
         disableCloseOnSelect
+        label={label}
```

Comment:
@pookmish after running AMP on the dev site, it still returned the error so poked around and realized that the resolution was to use either `label` or `title` attribute. 🤦 

Would you be able to refresh the dev site theme so that I can run AMP once more for good measure?

Reply:
Did that work?

Reply:
It worked! No more Severity 10! This PR should be good to go @pookmish 🎉